contracts/test/BasicSafe.vy

```diff
@@ -0,0 +1,13 @@
+# @version 0.3.7
```

Comment:
Just added this contract because deploying a gnosis safe for testing was proving difficult, but having a gnosis safe would be ideal because it would mimic production better. I think this serves well enough to test that `raw_call` fixes the error though.